app/src/marketplace/components/ProductPage/EditorNav/editorNav.pcss

```diff
@@ -0,0 +1,96 @@
+.root {
+  position: relative;
+}
+
+.EditorNav {}
+
+.progressBar {
+  position: absolute;
+  top: 15px;
+  bottom: 15px;
+  right: 0;
+  z-index: 0;
+  width: 16px;
+}
+
+.baseTrack {
+  position: absolute;
+  width: 1px;
+  background-color: #E7E7E7;
+  left: 50%;
+  top: 0;
+  z-index: 0;
+  height: 100%;
+  transform: translateX(-50%);
+}
+
+.progressTrack {
+  position: absolute;
+  width: 1.5px;
+  background-color: #0324FF;
+  left: 50%;
+  transform: translateX(-50%);
+  z-index: 1;
+  height: 0;
+  transition: height 0.2s;
+}
+
+.title {
```

Comment:
Could add `white-space: nowrap;` here to avoid this:

<img width="467" alt="Screenshot 2019-10-15 at 9 45 17" src="https://user-images.githubusercontent.com/1593398/66806726-e9b60680-ef30-11e9-82ad-5884cc3eddb8.png">